src/core/issues-notes.js

```diff
@@ -118,7 +118,20 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (report.title && githubAPI) {
+          const labelsGroup = Array.from(ghIssue.labels)
+            .map(createLabels)
+            .reduce((frag, labelElem) => {
+              frag.appendChild(labelElem);
+              return frag;
+            }, document.createDocumentFragment());
+          $tit.append(
+            $(
+              "<span style='text-transform: none'>: " + report.title + "</span>"
```

Comment:
This I had copied from the existing code, so in that this was inline styling. So, if you want me to make a class for this then I would have to make a class for this. If yes, suggest me a name for this class.

Reply:
if this is a `.issue`, then a `.issue .title, .note .title` rule would be good.